nixos/modules/hardware/cpu/intel-peg.nix

```diff
@@ -0,0 +1,86 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.hardware.cpu.intel.pegCpu;
+
+  machineConfigs = {
+    dell_latitude_e6x00 = {
+      link = "https://wiki.archlinux.org/index.php/Dell_Latitude_E6x00";
+      read = "44:32 0xCE";
```

Comment:
Maybe you could explain (or at least link to something that explains) how to figure out which values to use for a particular machine.